pkg/virt-launcher/virtwrap/device/hostdevice/generic/hostdev.go

```diff
@@ -28,7 +28,8 @@ import (
 )
 
 const (
-	AliasPrefix = "hostdevice-"
+	AliasPrefix       = "hostdevice-"
+	DefaultDisplayOff = false
```

Comment:
I see the point of this, i.e. avoiding to to pass cryptic booleans around, right?
However, this reads quite confusing to me, as it sounds like we want the display to not be off...
Maybe only keep "DefaultDisplayOn" like below and pass `!DefaultDisplayOn` to `CreateMDEVHostDevices()`?
I'm definitely nit-picking here, feel free to ignore, it's just something that I run into every so often, so I wonder what the best thing to do is.

Reply:
This is just a variable that holds a value.
When you read the implementation I think it will make more sense to you.

```
mdevHostDevices, err := hostdevice.CreateMDEVHostDevices(hostDevicesMetaData, mdevPool, DefaultDisplayOn)
```
and 
```
mdevHostDevices, err := hostdevice.CreateMDEVHostDevices(hostDevicesMetaData, mdevPool, DefaultDisplayOff)
```